deploy/olm-catalog/portworx/23.12.0/portworx-certified.clusterserviceversion.yaml

```diff
@@ -0,0 +1,544 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: portworx-operator.v23.12.0
+  namespace: placeholder
+  annotations:
+    capabilities: Auto Pilot
+    categories: "Storage"
+    description: Cloud native storage solution for production workloads
+    containerImage: registry.connect.redhat.com/portworx/openstorage-operator@sha256:6c0d870d8573747f79cbebf41614258326131173668aa7be5638c8dd7846ea54
```

Comment:
*`portworx/px-operator:23.12.0-dev`

Same for the two instances of the image below.

Reply:
Oh.. thanks!